scripts/run-artifactory.sh

```diff
@@ -40,3 +43,24 @@ docker cp ${SCRIPT_DIR}/artifactory-1.crt ${CONTAINER_ID_2}:/opt/jfrog/artifacto
 docker cp ${SCRIPT_DIR}/artifactory-2.crt ${CONTAINER_ID_1}:/opt/jfrog/artifactory/var/etc/access/keys/trusted/artifactory-2.crt
 
 echo "Circle-of-Trust is setup between artifactory-1 and artifactory-2 instances"
+
+echo "Generate Admin Access Keys for both instances"
+
+ARTIFACTORY_URLS=("${ARTIFACTORY_URL_1}" "${ARTIFACTORY_URL_2}")
+for ARTIFACTORY_URL in "${ARTIFACTORY_URLS[@]}";
+  do
+    COOKIES=$(curl -c - "${ARTIFACTORY_URL}/ui/api/v1/ui/auth/login?_spring_security_remember_me=false" \
```

Comment:
Can we call the `get-access-key.sh` here?

Reply:
yup, done

Reply:
Did you see the PR I created?

Reply:
I've copied almost everything from there, see my last commit. Thanks!